From a0a6c030edf233316b9acc56224bfc0d8f637308 Mon Sep 17 00:00:00 2001
From: abelino <abelino.romo@gmail.com>
Date: Tue, 26 Nov 2024 16:49:14 -0800
Subject: [PATCH] Prepare bootstrap file

The bootstrap script uses git to create a VERSION file and we do not
have access to any git history when pulling the tarball from GitHub's
Archive. Therefore, we move the responsibility of generating the
VERSION file and autoreconf'ing from the bootstrap script and off to
this package's makefile.

Signed-off-by: abelino <abelino.romo@gmail.com>
---
 bootstrap | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/bootstrap b/bootstrap
index 47135577..2032dde8 100755
--- a/bootstrap
+++ b/bootstrap
@@ -1,8 +1,6 @@
 #!/bin/sh
 set -e
 
-git describe --tags --always --dirty > VERSION
-
 # generate list of source files for use in Makefile.am
 # if you add new source files, you must run ./bootstrap again
 src_listvar () {
@@ -89,11 +87,3 @@ if test "${GEN_FUZZ}0" -eq 10; then
 else
   touch Makefile-fuzz-generated.am
 fi
-
-${AUTORECONF} --install --sym $@
-
-if grep "Invalid policy. Valid policies: git-directory, minor-version." configure >/dev/null; then
-    echo "ERROR: ax_is_release.m4 is outdated. ./configure will fail."
-    echo "Please download from http://ftpmirror.gnu.org/autoconf-archive/autoconf-archive-2019.01.06.tar.xz"
-    exit 1
-fi
-- 
2.47.0

